content/docs/guides/overview.md

```diff
@@ -50,30 +50,34 @@ Gherkin serves multiple purposes:
 The Cucumber grammar exists in different flavours for many [spoken languages](/docs/gherkin/reference#spoken-languages)
 so that your team can use the keywords in your own language.
 
-Gherkin documents are stored in `.feature` text files and are typically versioned in source control
-alongside the software.
+Gherkin documents are stored in `.feature` text files and are typically
+versioned in source control alongside the software.
 
 See the [Gherkin reference](/docs/gherkin) for more details.
 
 {{% note "Who should write Gherkin?" %}}
-It's usually best to let developers write Gherkin if the team is practicing BDD (test first).
+It's usually best to let developers write Gherkin if the team is practicing BDD
+(test first).
 
-If Cucumber is used solely as a test automation tool (test after) it can be done by
-testers or developers.
+If Cucumber is used solely as a test automation tool (test after) it can be done
+by testers or developers.
 
-It is usually counterproductive to let product owners and business analysts write Gherkin.
-Instead, we recommend they participate in [Example Mapping](/docs/bdd/example-mapping) sessions
-and **approve** the Gherkin documents after a developer or tester has translated it to Gherkin.
+It is usually counterproductive to let product owners and business analysts
+write Gherkin. Instead, we recommend they participate in [Example Mapping](/docs/bdd/example-mapping)
+sessions and **approve** the Gherkin documents after a developer or tester has
+translated it to Gherkin.
 {{% /note %}}
 
-# Step Definitions
+# What are Step Definitions?
 
-In addition to [feature files](/docs/gherkin/reference#feature), Cucumber needs a set of [step definitions](/docs/cucumber/step-definitions). Step definitions map (or "glue") each
-Gherkin step to programming code to carry out the  action that should be performed by the step.
+[Step definitions](/docs/cucumber/step-definitions) connect Gherkin steps to
+programming code. A step definition carries out the action that should be
+performed by the step. So step definitions hard-wire the specification to the
+implementation.
 
-Step definitions hard-wire the specification to the implementation.
-
-<!-- TODO: Illustration (Feature) - (Step Defs) -> (System) -->
+```
+ | Steps in Gherkin | --matched with--> | Step Definitions | --manipulates--> | System |
+```
```

Comment:
```
┌────────────┐                 ┌──────────────┐                 ┌───────────┐
│   Steps    │                 │     Step     │                 │           │
│ in Gherkin ├──matched with──▶│ Definitions  ├───manipulates──▶│  System   │
│            │                 │              │                 │           │
└────────────┘                 └──────────────┘                 └───────────┘
```

I opened Monodraw, and had so much fun drawing this.

Reply:
Thanks! Much appreciated!